pkg/tool/git.go

```diff
@@ -72,5 +72,6 @@ func (g Git) GetUrlForRemote(remote string) (string, error) {
 }
 
 func (g Git) ValidateRepository() error {
-	return g.exec.RunProcess("git", "rev-parse", "--is-inside-work-tree")
+	_, err := g.exec.RunProcessAndCaptureOutput("git", "rev-parse", "--is-inside-work-tree")
+	return err
```

Comment:
Is this functionally any different from what was implemented before?

Reply:
The difference is that it doesn't log anything. The output of `git rev-parse --is-inside-work-tree` is `true`, which was logged but didn't make much sense as such.